doc/examples/reconst_dki.py

```diff
@@ -228,19 +223,62 @@
    DKI (upper panels) and DTI (lower panels).**.
 
-From the figure, we can see that the DT standard diffusion measures from DKI
-are noisier than the DTI measurements. This is a well known pitfall of DKI
-[NetoHe2014]_. Since it involves the estimation of a larger number of
-parameters, DKI is more sensitive to noise than DTI. Nevertheless, DT diffusion
-based measures were shown to have better precision (i.e. less sensitive to
-bias) [Veraa2011]_.
+From the figure, we can see that the standard diffusion measures of the
+HCP-like data obtain from the diffusion kurtosis model have similar contracts
+to the standard diffusion measures obtain from the tensor model. This can be
+explained by the high quality of the dataset and the high preformance of the
+diffusion kurtosis fit procedure.
+
+In addition to the standard diffusion statistics, the DiffusionKurtosisFit
+instance can be used to estimate the non-Gaussian measures of mean kurtosis
+(MK), the axial kurtosis (AK) and the radial kurtosis (RK).
+"""
+
+MK = dkifit.mk(0, 3)
+AK = dkifit.ak(0, 3)
+RK = dkifit.rk(0, 3)
+
+"""
+Kurtosis measures are susceptible to high amplitude outliers. The impact of
+high amplitude kurtosis outliers were removed on the above lines of codes by
+introducing as an optional input the extremes of the typical values of kurtosis
+(assumed here as the values on the range between 0 and 3)
 
-The standard kurtosis statistics can be computed from the DiffusinKurtosisFit
-instance as the mean kurtosis (MK), the axial kurtosis (AD) and the radial
-kurtosis (RK).
+Now we are ready to plot the kurtosis standard measures using matplotlib:
+"""
+
+fig2, ax = plt.subplots(1, 3, figsize=(12, 6),
+                        subplot_kw={'xticks': [], 'yticks': []})
+
+fig2.subplots_adjust(hspace=0.3, wspace=0.05)
+
+ax.flat[0].imshow(MK[:, :, axial_slice], cmap='gray')
+ax.flat[0].set_title('MK')
+ax.flat[1].imshow(AK[:, :, axial_slice], cmap='gray')
+ax.flat[1].set_title('AK')
+ax.flat[2].imshow(RK[:, :, axial_slice], cmap='gray')
+ax.flat[2].set_title('RK')
+
+plt.show()
+fig2.savefig('Kurtosis_tensor_standard_measures.png')
+
+"""
+.. figure:: Kurtosis_tensor_standard_measures.png
+   :align: center
+   **Kurtosis tensor standard measures obtain from the kurtosis tensor.**.
+
+The non-Gaussian behaviour of the diffusion signal is larger when water
+diffusion is restrited by compartments and barriers (e.g., myelin sheath).
+Therefore, as the figure above shows, white matter kurtosis values are smaller
+along the axial direction of fibers (smaller amplitudes shown in the AK map)
+than for the radial directions (larger amplitudes shown in the RK map).
 
 References:
 
+.. [TaxCMW2015] Tax CMW, Otte WM, Viergever MA, Dijkhuizen RM, Leemans A
+                (2014). REKINDLE: Robust extraction of kurtosis INDices with
+                linear estimation. Magnetic Resonance in Medicine 73(2):
+                794–808.
```

Comment:
FYI: I indeed had this (specific to Windows?), solved it by putting 
```
#!/usr/bin/env python
# -*- coding: utf-8 -*-
```
at the start of the file, but indeed better to prevent it :)